libs/ng-utils/src/lib/directives/formcontrol-augmentation.directive.spec.ts

```diff
@@ -0,0 +1,36 @@
+import { CommonModule } from '@angular/common';
+import { FormControl, ReactiveFormsModule } from '@angular/forms';
+import { createDirectiveFactory, SpectatorDirective } from '@ngneat/spectator';
+
+import { FormControlAugmentationDirective } from './formcontrol-augmentation.directive';
+
+describe('Directive: FormControlAugmentation', () => {
+  let spectator: SpectatorDirective<FormControlAugmentationDirective>;
+  let directive: FormControlAugmentationDirective;
+
+  const createDirective = createDirectiveFactory({
+    directive: FormControlAugmentationDirective,
+    imports: [CommonModule, ReactiveFormsModule],
+  });
+
+  beforeEach(() => {
+    const testControl = new FormControl(null);
+
+    spectator = createDirective(
+      `
+        <input [formControl]="control" #templateVariableName="formControl"/>
+    `,
+      {
+        hostProps: {
+          control: testControl,
+        },
+      }
+    );
+
+    directive = spectator.directive;
+  });
+
+  it('should create', () => {
+    expect(directive).toBeDefined();
+  });
```

Comment:
Add test to see if the #templateVariableName hold the expected value exported from the directive.

Reply:
I've tried, I haven't found a way to extract the template variable value here in the unit test, but I'll take another look

Reply:
Done